stable/efs-provisioner/README.md

```diff
@@ -0,0 +1,106 @@
+# Helm chart for 'efs-provisioner'
+
+The Kubernetes project provides an AWS [EFS provisioner](https://github.com/kubernetes-incubator/external-storage/tree/master/aws/efs) 
+that is used to fulfill PersistentVolumeClaims with EFS PersistentVolumes.
+
+"The efs-provisioner allows you to mount EFS storage as PersistentVolumes in kubernetes. 
+It consists of a container that has access to an AWS EFS resource. The container reads 
+a configmap which contains the EFS filesystem ID, the AWS region and the name you want 
+to use for your efs-provisioner. This name will be used later when you create a storage class."
+
+This chart deploys the EFS Provisioner and a StorageClass for EFS volumes (optionally as the default).
+
+The EFS external storage provisioner runs in a Kubernetes cluster and will create persistent volumes
+in response to the PersistentVolumeClaim resources being created. These persistent volumes can then be
+mounted on containers.
+
+  https://kubernetes.io/docs/concepts/storage/persistent-volumes/
+
+The persisent volumes are created as folders with in an AWS EFS filesystem.
+
+  https://aws.amazon.com/efs/
+
+## Prequisites
+
+You must create the EFS file system and end points yourself first.
+
+  https://docs.aws.amazon.com/efs/latest/ug/creating-using-create-fs.html
+
+The end points must be accessible to the cluster and the cluster nodes must have
+permission to mount EFS file systems.
+
+## Configuring
+
+At a minimum you must the supply the EFS file system ID and the AWS region
+
+```
+helm install incubator/efs-provisioner --set efsFileSystemId=fs-12345678 --set awsRegion=us-east-2
```

Comment:
Since you've moved to stable, can you please update the readme?

Reply:
Ah yes. Good point @jeyglk. I'll do that and search for other refs.

Reply:
Done 👍